### ISSUE TITLE
Landmarks on the Learn page

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#9623]
_Follow up issues to be opened to cover Coach, Facility and Device pages._

## Observed behavior
We need to improve the landmarks on the **Learn** page to provide more accessible page navigation options. The _main_ 🙂 issue is the position of the ``main`` element, or the **div** with the ``role="main"`` which is used for skipping to main content of the page. Ideally, this element should not include and come after the sub-page navigation element.

### Learn > Home
Current element with the ``role="main"`` is almost the topmost **div**, and does not serve well the purpose of skipping content. Should be the **div** that contains only the main heading and the sections below.

![LEDev (start)  Running  - Oracle VM VirtualBox_004](https://user-images.githubusercontent.com/1457929/184784650-b7d8e0a6-3eab-4fe4-9867-60b1eafbc3b6.png)

#### Learn > Home > Classes & Lessons
Same as above for the current element with the ``role="main"``. Should be the **div** that contains the breadcrumbs, the main heading and the sections below.

Class | Lesson
--- | ---
![LEDev (start)  Running  - Oracle VM VirtualBox_005](https://user-images.githubusercontent.com/1457929/184785630-ddf042e2-2b3d-4cc6-9fdd-f511795d53f2.png) | ![LEDev (start)  Running  - Oracle VM VirtualBox_006](https://user-images.githubusercontent.com/1457929/184785635-54025125-d659-4dea-b3f8-9118b47855c4.png) 

More about  ``role="navigation"`` below.

#### Learn > Home > Quizzes
Quizzes open in a separate page without the appbar, so there wouldn't be much to skip, and it is probably not necessary to reposition the  ``role="main"``. Bonus would be to wrap the list a questions in a **div** with ``role="navigation"`` and add the ``aria-label="Jump to question"``.

Quiz | Questions
--- | ---
![LEDev (start)  Running  - Oracle VM VirtualBox_007](https://user-images.githubusercontent.com/1457929/184787685-f9082fd3-789b-4283-b9fc-6ec50f72ca5f.png) | ![LEDev (start)  Running  - Oracle VM VirtualBox_017](https://user-images.githubusercontent.com/1457929/184787684-aaf6dfa9-6ec1-4c21-bbc3-b5af60590072.png)

#### Content renderers
Here we have one extra ``role="main"`` that should be deleted.

Video/Audio | HTML
--- | ---
![LEDev (start)  Running  - Oracle VM VirtualBox_009](https://user-images.githubusercontent.com/1457929/184788466-d87e4c45-0d65-499f-a7c3-de6a2987c8d0.png) | ![LEDev (start)  Running  - Oracle VM VirtualBox_010](https://user-images.githubusercontent.com/1457929/184788486-0a3a02a3-d21e-4a59-9a75-c3a000244ce5.png)

### Learn > Library
The ``main`` element seems more appropriate, so the ``role="main"`` of the container **div** should be deleted. 
To make the sidebar more accessible, we should add a ``role="region"`` and add the ``aria-label="Filter and search"``.

Channels | Sidebar
--- | ---
![LEDev (start)  Running  - Oracle VM VirtualBox_011](https://user-images.githubusercontent.com/1457929/184789674-425e58f2-7012-4707-b2dd-fee09cde8e64.png) | ![LEDev (start)  Running  - Oracle VM VirtualBox_012](https://user-images.githubusercontent.com/1457929/184789690-00c46ad4-76ac-40f9-9055-9ee8f4ed4b1e.png)

When the screen size is such that the sidebar opens in front, it should also have the ``role="region"`` and ``aria-label="Filter and search"``.

![LEDev (start)  Running  - Oracle VM VirtualBox_014](https://user-images.githubusercontent.com/1457929/184790106-7af0f9b9-2109-4549-9576-6e9c7b3bcd67.png)

### Learn > Bookmarks
Same as for **Learn > Home**: current element with the ``role="main"`` is almost the topmost **div**. Should be the **div** that contains only the main heading and the cards below.

![LEDev (start)  Running  - Oracle VM VirtualBox_015](https://user-images.githubusercontent.com/1457929/184790363-1e010218-03fe-4104-b088-c88f6542dc0c.png)

### Navigation sections
There may be 2 or 3 on a given page, so we need to label them properly.
1. Main sidebar navigation has the proper role, but I'm thinking that the ``aria-label`` should be changed into ``Main user menu``. The word **menu** is more common and less technical than **navigation**.
2. Sub-page navigation is inside the ``nav`` element but would need the ``aria-label="Learn page menu"``.
3. Breadcrumbs menu at **Learn > Home > Classes** is also inside the ``nav`` element, and I would suggest the ``aria-label="Classes and assignments"``. 

![LEDev (start)  Running  - Oracle VM VirtualBox_016](https://user-images.githubusercontent.com/1457929/184791417-22697d24-f6a9-4d6e-85f6-db3ae46efba8.png)

## References and tools
Recomended reading: http://matatk.agrip.org.uk/talks/2020/landmarks/
Browser extension/addon: http://matatk.agrip.org.uk/landmarks/

## Errors and logs
…

## Expected behavior
Proper page landmarks with labels when required.

## User-facing consequences
Landmarks will provide more accessible user experience (ref. [WCAG SC 2.4.1 - Bypass Blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks))

## Steps to reproduce
…

## Context
 * Kolibri version: current ``develop``
 * Operating system: all
 * Browser: all

**Comment by radinamatic** [comment]
cc @marcellamaki
